src/SymptomChecker/AtRiskRecommendation.tsx

```diff
@@ -71,8 +112,32 @@ const style = StyleSheet.create({
   cancelButtonInnerContainer: {
     padding: Spacing.medium,
   },
+  image: {
+    resizeMode: "contain",
+    width: "97%",
+    height: 300,
```

Comment:
Does this fixed height force a scroll on small format screens?

Reply:
Yeah it does, I'll make this smaller and reduce the text size to avoid scrolling shenanigans.